sdk/src/asset_handlers/pdf_io.rs

```diff
@@ -235,22 +232,10 @@ pub mod tests {
     }
 
     #[test]
-    fn test_read_cai_returns_cai_bytes() {
-        let source = include_bytes!("../../tests/fixtures/basic.pdf");
-
-        let mut pdf = Pdf::from_bytes(source).unwrap();
-        assert!(pdf.read_manifest_bytes().unwrap().is_none());
-
-        let mut pdf_with_manifest = vec![];
-        let expected_manifest = vec![0, 1, 1, 2, 3, 5, 8, 13, 21, 34];
-
-        pdf.write_manifest_as_annotation(expected_manifest.clone())
-            .unwrap();
-        pdf.save_to(&mut pdf_with_manifest).unwrap();
-
+    fn test_read_cai_on_express_pdf_finds_manifest() {
```

Comment:
Can we rename this to test_read_cai_express_pdf_finds_manifest to test_read_cai_express_pdf_finds_single_manifest_store since we will eventually have the multiple manifest stores

Reply:
That makes sense, will do!